manual/query_timestamps/README.md

```diff
@@ -59,6 +59,48 @@ session.execute(statement);
 
 [tsg]: http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/core/TimestampGenerator.html
 
+### Overview of Timestamp Generators
+
+The driver ships with two different implementations of [TimestampGenerator][tsg]:
+
+1. [AtomicMonotonicTimestampGenerator][amtsg], which guarantess monotonicity of timestmaps for all threads;
+2. [ThreadLocalMonotonicTImestampGenerator][tlmtsg], which guarantees per-thread monotonicity of timestamps.
```

Comment:
timestmaps => timestamps

ThreadLocalMonotonicTImestampGenerator => ThreadLocalMonotonicT**_i_**mestampGenerator (also in the referenced URL)